src/main/java/programminglife/model/drawing/DrawableNode.java

```diff
@@ -1,198 +1,134 @@
 package programminglife.model.drawing;
 
-import javafx.scene.paint.Color;
 import javafx.scene.shape.Rectangle;
-import programminglife.model.*;
+import programminglife.model.GenomeGraph;
+import programminglife.model.Link;
+import programminglife.model.XYCoordinate;
 
 import java.util.Collection;
-import java.util.LinkedHashSet;
-import java.util.NoSuchElementException;
 
 /**
- * A {@link Segment} that also Implements {@link Drawable}.
+ * Created by toinehartman on 12/06/2017.
```

Comment:
IntelliJ should give a warning about this. Might be better not to use the standard javadoc